core/src/main/java/io/ddf/content/SqlTypedResult.java

```diff
@@ -0,0 +1,76 @@
+package io.ddf.content;
+
+
+import java.sql.SQLType;
```

Comment:
this interface is added in Java 8 but our build and runtime Java version is only 7.x atm